torch/_prims/__init__.py

```diff
@@ -1523,7 +1523,7 @@ def _slice_meta(
 
     new_shape = []
     for x, y, z in zip(start_indices, limit_indices, _strides):
-        new_shape.append(math.floor((y - x) / z))
```

Comment:
Even if this formula was mathematically correct, it would have failed for `x, y, z = 0, 2**53 + 1, 1`! 👀